data/ecosystem/distributions.yaml

```
@@ -75,6 +75,10 @@
   url: https://github.com/microsoft/ApplicationInsights-Java
   docsUrl: https://learn.microsoft.com/azure/azure-monitor/app/opentelemetry-enable?tabs=java
   components: [Java]
+- name: Azure Monitor OpenTelemetry Distro / Application Insights in Spring Boot native image Java application
```

```suggestion
- name: Azure Monitor OpenTelemetry Distro
```

Any chance we could shorten it to the above?

---

For the Microsoft distribution of the OTel Java agent, the name is `Azure Monitor Distribution of OpenTelemetry Java`.

What do you think about `Azure Monitor Distribution of OpenTelemetry Java Spring Boot starter`, or `Azure Monitor Distribution of OpenTelemetry Java starter`?

Other vendors providing a distribution of the OpenTelemetry Spring Boot starter could follow the same pattern.

---

Suggesting the following pattern:

- Azure Monitor Distribution of OpenTelemetry Java (Spring Boot starter)

---

Great! I have updated the PR with this formulation.